src/MICore/MICore.csproj

```diff
@@ -199,7 +200,8 @@
   <ItemGroup>
     <GlassDirCopy Include="$(OutDir)$(AssemblyName)$(TargetExt)" />
     <GlassDirCopy Include="$(OutDir)$(AssemblyName).XmlSerializers.dll" Condition="'$(Configuration)'!='Debug-PortablePDB'" />
-    <GlassDirCopy Include="$(OutDir)Newtonsoft.Json.dll" />
+	<GlassDirCopy Include="$(NuGetPackagesDirectory)\Newtonsoft.Json\6.0.8\lib\portable-net45+wp80+win8+wpa81+aspnetcore50\Newtonsoft.Json.dll" Condition="'$(IsCoreClr)' == 'true'" />
```

Comment:
Do you need `Condition="'$(IsCoreClr)' == 'xyz'"`? 
Shouldn't this just copy the binary if
`<GlassDirCopy Include="$(NuGetPackagesDirectory)\Newtonsoft.Json\6.0.8\lib\portable-net45+wp80+win8+wpa81+aspnetcore50\Newtonsoft.Json.dll" /> ` and remove the duplicate.